test/client/fixtures/sandbox/code-instrumentation/location-test.js

```diff
@@ -192,6 +192,204 @@ test('special pages (GH-339)', function () {
     destLocation.forceLocation(storedForcedLocation);
 });
 
+test('should process some url types in locationWrapper (href, replace, assign) (GH-1613)', function () {
```

Comment:
`test('different url types for locationWrapper methods (href, replace, assign) (GH-1613)'`